opendoors/mysql.py

```diff
@@ -70,8 +70,12 @@ def read_table_to_dict(self, database: str, tablename: str):
         :return: The contents of the table as a dict.
         """
         cursor = self.conn.cursor()
-        cursor.execute(f"SELECT * FROM {database}.{tablename};")
-        return cursor.fetchall()
+        try:
+            cursor.execute(f"SELECT * FROM {database}.{tablename};")
+            return cursor.fetchall()
+        except Exception as e:
+            self.logger.info(f"No table {tablename} in {database}...")
+            return []
```

Comment:
Maybe let's change `return []` to `return {}` to always return a dict, instead of returning an empty list when the table does not exist?